components/ui/Accordion/index.tsx

```diff
@@ -23,11 +23,46 @@ export default function Accordion(props: Props): ReactElement {
   const [isExpanded, setIsExpanded] = useState(true);
   const [height, setHeight] = useState(`${content?.current?.scrollHeight}px`);
   const [loaded, setLoaded] = useState(false);
+
   const handleExpand = () => {
     setIsExpanded(!isExpanded);
     setHeight(isExpanded ? '0px' : `${content?.current?.scrollHeight}px`);
   };
   const svgClasses = classNames('w-3 h-3 fill-current mb-1 mr-2');
+
+  useEffect(() => {
+    const buttons = window?.document.querySelectorAll(
+      '.showExternalVideoButton'
+    );
+    const container = window?.document.getElementById(id + 'container')!;
+
+    const handleNewHeight = (e: Event) => {
+      const oldHeight = Number(container?.style.height.slice(0, -2));
+      if (
+        document.querySelector(`#${id}container .showExternalVideoButton`) !==
+        null
+      ) {
+        const target = e.currentTarget as HTMLButtonElement;
+        // adding the height of the video (500|240) - the height of the dissapearing container (185.5) = 314.5|54.5
```

Comment:
Also shouldn't `(500|240) - the height` be `(500|240) + the height` since we are adding? unless I'm missing something?

Reply:
When I am talking about the "container" I am refering to the buttons component in which the user agrees to show the video. So when the user clicks, technically that box dissappears (it has a height of 185.5), but the video has a height of 500 or 240. So that is why I am refering to it as  adding (500|240 - 185.5).  I could give it a more of a descriptive name than "container".